fmriprep/interfaces/maths.py

```diff
@@ -0,0 +1,48 @@
+import os
+import numpy as np
+from nipype.interfaces.base import SimpleInterface, TraitedSpec, traits, File
+from nipype.utils.filemanip import fname_presuffix
+
+
+class ClipInputSpec(TraitedSpec):
+    in_file = File(exists=True, mandatory=True, desc="Input imaging file")
+    out_file = File(desc="Output file name")
+    minimum = traits.Float(-np.inf, usedefault=True,
+                           desc="Values under minimum are set to minimum")
+    maximum = traits.Float(np.inf, usedefault=True,
+                           desc="Values over maximum are set to maximum")
+
+
+class ClipOutputSpec(TraitedSpec):
+    out_file = File(desc="Output file name")
+
+
+class Clip(SimpleInterface):
+    """ Simple clipping interface that clips values to specified minimum/maximum
+
+    If no values are outside the bounds, nothing is done and the in_file is passed
+    as the out_file without copying.
+    """
+    input_spec = ClipInputSpec
+    output_spec = ClipOutputSpec
+
+    def _run_interface(self, runtime):
+        import nibabel as nb
+        img = nb.load(self.inputs.in_file)
+        data = img.get_fdata()
```

Comment:
won't this potentially convert data to `float64`?

Reply:
Sure, but it's only one volume at a time. It will not save the outputs as float64, since we preserve the header.

Reply:
ah, then it's fine as is